.github/workflows/conversion_script_test.yaml

```diff
@@ -62,6 +62,7 @@ jobs:
             aws s3 cp s3://pytorch-multimodal/llama2-7b/consolidated.00.pth /tmp/test-artifacts/llama2-7b-fair
       - name: Install dependencies
         run: |
+          python -m pip install torch
```

Comment:
Yeah this part I have not yet taken a stance on here. In practice, we can just add different versions to the matrix. In my experience running CI on nightlies means it will be red decently often. If we are OK with that I am happy to do it, but personally wanna discuss with folks a bit more on the pros and cons here.

Reply:
I guess it would depend on whether we gate on any nightly features? I think ultimately we won't have an option, but would love to see if we can hold off for as long as possible. The only caveat to this is that we also want to catch pushes to torch which break us.